src/check-file-exists-at-url.ts

```diff
@@ -0,0 +1,19 @@
+import https from 'https';
+
+export function checkFileExistsAtUrl(url: string): Promise<boolean> {
+  return new Promise((resolve, reject) => {
+    const urlParts = new URL(url);
+    const {
+      hostname,
+      pathname: path,
+    } = urlParts;
+
+    const request = https.get(url);
+
+    request.on('response', ({statusCode}) => {
+      resolve(statusCode === 200);
```

Comment:
Note... I'm taking a shortcut here and ignoring the contents. We could just load the headers and kill execution but it's probably more work than it's worth.